pyramid_mixpanel/consumer.py

```diff
@@ -20,18 +20,25 @@ class MockedMessage:
     msg: t.Dict["str", object]
 
 
-@dataclass(frozen=True)
+@dataclass
 class MockedConsumer:
     """Save messages in an internal list, useful in unit testing."""
 
     # Internal storage of mocked message
     mocked_messages: t.List = field(default_factory=lambda: [])
 
+    # Flag if .flush() was called
+    flushed: t.Optional[bool] = None
```

Comment:
Since this is `bool` would it make sense to have default value also boolean, like say "False" ?

Reply:
This makes sense in a very tiny percentage of use cases. This one just might be one of those. I'll try and see if I like it.